pkg/build/gobuild.go

```diff
@@ -255,12 +255,17 @@ func build(ctx context.Context, ip string, dir string, platform v1.Platform, con
 		return "", fmt.Errorf("could not create env for %s: %w", ip, err)
 	}
 
-	tmpDir, err := ioutil.TempDir("", "ko")
+	tmpDir, err := ioutil.TempDir(os.TempDir(), "ko")
 	if err != nil {
 		return "", err
 	}
 
 	if dir := os.Getenv("KOCACHE"); dir != "" {
+		if _, err := os.Stat("dir"); os.IsNotExist(err) {
```

Comment:
This should be `dir`, not the string `"dir"`.

Reply:
thank you, good catch 🙈